.clang-format

```diff
@@ -0,0 +1,173 @@
+Language:        Cpp
+IndentWidth:     2
+UseTab:		Never
+ColumnLimit:     120
+SortIncludes:    false
+AccessModifierOffset: -2
+ReflowComments:  false
+ConstructorInitializerIndentWidth: 2
+ContinuationIndentWidth: 2
+AllowShortFunctionsOnASingleLine: None
+BreakBeforeBraces: Custom
+BraceWrapping:
+    AfterExternBlock: true
+    AfterClass: true
+    AfterFunction: true
+    AfterNamespace: false
+    AfterEnum: false
+    AfterCaseLabel: false  
```

Comment:
'AfterCaseLabel: true' better matches the current style.

I rather dislike that the `break` behind a closing brace is put onto the same line, but found no option to influence this. The developers of clang-format seem to believe that this is as it should be.